src/lib.rs

```diff
@@ -35,22 +35,25 @@ pub async fn app(config: &Figment) -> Result<Rocket<Build>> {
         ));
     }
 
+    let mut routes = routes![
+        put_content,
+        batch_put_content,
+        delete_content,
+        open_orbit_allowlist,
+        open_orbit_authz,
+        cors
+    ];
+
+    if kepler_config.public_get {
+        let mut no_auth = routes![get_content_no_auth, list_content_no_auth];
+        routes.append(&mut no_auth);
+    } else {
+        let mut auth = routes![get_content, list_content];
```

Comment:
Could it make sense to have those always? I can think of a feature where that would make sense: some files are private. (Maybe access logs as well.) Otherwise I think we can remove Rocket's `forward` bit in the authentication.

Reply:
it would make sense, but I was unable to make the `rank`s work in the right way. Will try a better `forward` usage.